devito/symbolics/extended_sympy.py

```diff
@@ -58,6 +59,10 @@ class CondEq(sympy.Eq, FrozenExpr):
     def __new__(cls, *args, **kwargs):
         return sympy.Eq.__new__(cls, *args, evaluate=False)
 
+    @property
+    def canonical(self):
```

Comment:
what's this one?

Reply:
stuff that SymPy needs due to some weird internal mechanics